code/modules/cargo/exports/large_objects.dm

```diff
@@ -88,20 +88,10 @@
 	export_types = list(/obj/item/gun/energy/minigun)
 
 /datum/export/large/fev/FEV_solution_one
-	cost = 10000
-	unit_name = "FEV prototype"
+	cost = 9000
+	unit_name = "FEV solution"
 	export_types = list(/obj/item/reagent_containers/glass/bottle/FEV_solution)
 
-/datum/export/large/fev/FEV_solution_two
-	cost = 10000
-	unit_name = "FEV mk2"
-	export_types = list(/obj/item/reagent_containers/glass/bottle/FEV_solution/two)
-
-/datum/export/large/fev/FEV_solution_curling
```

Comment:
Significantly more dangerous. No incentive to ship it?

Reply:
Sure there is, the code did not work previously. It was stuck on only giving 10000 caps, which was a bit overturned. And all the FEV was shipped in the shuttle.
9000 works a lot better when the balance of the increased material export gets taken into consideration.